apps/website/src/content/docs/anchor.mdx

```diff
@@ -31,6 +42,26 @@ The anchor component supports the polymorphic `as` prop to render the anchor as
 
 The `as` prop can also be used with `Link` components from different routers such as [Next.js](https://nextjs.org/docs/api-reference/next/link) and [React Router](https://reactrouter.com/en/main/components/link)
 
+### External links
+
+When linking to an external site, you can use the `isExternal` prop to indicate that the link will take the user to a different domain. This will add a standard icon to the end of the link text.
+
+<LiveExample src='Anchor.external.jsx' truncate={false}>
+  <AllExamples.AnchorExternalExample client:load />
+</LiveExample>
```

Comment:
I'm not fully sure about this documentation. I feel like the icon is only useful when links are opening in a new tab. Is there any benefit to showing the icon when not specifying `target=_blank`?

At the same time, I do not want to encourage users to blanket add `isExternal target='_blank'` everywhere. That's why I kept `isExternal` and `target=_blank` separate from each other, but can't find a nice way to document it.

Reply:
> I'm not fully sure about this documentation. I feel like the icon is only useful when links are opening in a new tab. Is there any benefit to showing the icon when not specifying `target=_blank`?

I too can't think of any case where the icon without the `target=_blank` is beneficial.

If there really isn't any benefit, should we consider showing the icon whenever we get `target=_blank`? Or maybe letting the users decide will be better to avoid unexpected visual changes for the user. 

> At the same time, I do not want to encourage users to blanket add `isExternal target='_blank'` everywhere. That's why I kept `isExternal` and `target=_blank` separate from each other, but can't find a nice way to document it.

I feel this documentation adequately separates `isExternal` and `target='_blank'`. So as-is is good imo.